ext/opcache/jit/zend_jit_ir.c

```diff
@@ -8511,22 +8522,20 @@ static int zend_jit_push_call_frame(zend_jit_ctx *jit, const zend_op *opline, co
 		used_stack_ref = ir_PHI_2(IR_ADDR, ref, used_stack_ref);
 	}
 
-	zend_jit_start_reuse_ip(jit);
-
 	// JIT: if (UNEXPECTED(used_stack > (size_t)(((char*)EG(vm_stack_end)) - (char*)call))) {
-	jit_STORE_IP(jit, ir_LOAD_A(jit_EG(vm_stack_top)));
+	rx = ir_LOAD_A(jit_EG(vm_stack_top));
```

Comment:
The variable `rx` is named after the `RX` register of the old JIT, which was an alias for r15 (IP): https://github.com/php/php-src/blob/81d9a27c47b12ad20e180afb09c6188c8d403f9f/ext/opcache/jit/zend_jit_x86.dasc#L30

It could be renamed to `call`, but this would increase the diff size.